src/tsdb.c

```diff
@@ -18,7 +18,7 @@
 static Series* lastDeletedSeries = NULL;
 
 Series *NewSeries(RedisModuleString *keyName, Label *labels, size_t labelsCount, uint64_t retentionTime,
-        short maxSamplesPerChunk)
+        short maxSamplesPerChunk, int uncompressed)
```

Comment:
Since it hasn't been used anywhere in the project.

Reply:
@ashtul FYI, You are using booleans in gorilla.c.

Reply:
I do not push bool type into existing code since it requires an `#include <stdbool.h>` and not everyone is a fan of it.
gorrila.c was written from scratch